assets/javascripts/discourse/templates/components/product-item.hbs

```diff
@@ -7,15 +7,14 @@
 {{#if isLoggedIn}}
   <div class="product-purchase">
     {{#if product.repurchaseable}}
+      {{#link-to "s.show" product.id class="btn btn-primary"}}
+        {{i18n 'discourse_subscriptions.subscribe.title'}}
+      {{/link-to}}
       {{#if product.subscribed}}
-        <span class="purchased">&#x2713; {{i18n 'discourse_subscriptions.subscribe.purchased'}}</span>
         {{#link-to "user.billing.subscriptions" currentUser.username class="billing-link"}}
-          {{i18n 'discourse_subscriptions.subscribe.go_to_billing'}}
+          {{i18n 'discourse_subscriptions.subscribe.view_past'}}
```

Comment:
Is the previous string (`go_to_billing`) used anywhere else in the plugin? It should be removed from `en` locale if it isn't.

Reply:
Yep! It's used here:

https://github.com/discourse/discourse-subscriptions/blob/3163e6be3161842d1fa327f2dd3dac8c53980053/assets/javascripts/discourse/templates/components/product-item.hbs#L22